bundle/manifests/topolvm.io_logicalvolumes.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Was not this CRD present earlier with a different name?

Reply:
Yes - this changes because of the name change.